monorepo/website/src/pages/index.astro

```diff
@@ -0,0 +1,142 @@
+                                </li>
+                                <li class='flex items-start gap-2'>
+                                    <span class='mt-1 h-1.5 w-1.5 flex-shrink-0 rounded-full bg-primary/70' aria-hidden='true'></span>
+                                    Browse sequences with powerful search and filtering options.
+                                </li>
+                                
+                            </ul>
+                        </div>
+                        <div class='hidden h-full w-px bg-primary/20 sm:block' aria-hidden='true'></div>
+                       
+                            {
+                                hasAggregateStats ? (
+ <div class='md:max-w-sm min-w-[12rem]'>
+                            <span class='text-xs font-semibold uppercase tracking-wide text-primary/80'>Dataset</span>
+                                    <div class='mt-4 grid grid-cols-2 gap-4 text-sm text-primary-900'>
+                                        <div>
+                                            <div class='text-xs uppercase tracking-wide text-primary-700'>Sequences</div>
+                                            <div class='text-xl font-semibold text-primary-800'>
+                                                {formatNumberWithDefaultLocale(aggregateSubmissionStats.total)}
+                                            </div>
+                                        </div>
+                                        <div>
+                                            <div class='text-xs uppercase tracking-wide text-primary-700'>Last {numberDaysAgoStatistics} days</div>
+                                            <div class='text-xl font-semibold text-primary-800'>
+                                                +{formatNumberWithDefaultLocale(aggregateSubmissionStats.recent)}
+                                            </div>
+                                        </div>
+                                    </div>
+  </div>
+                                ) : null
+                            }
+                      
+                    </div>
+                </div>
+            </div>
+            {
+                latestNewsItem && (
+                    <aside class='flex h-full flex-col justify-between rounded-2xl border border-base-200/70 bg-base-100 p-6 shadow-[0_4px_12px_rgba(0,0,0,0.05)]'>
+                        <div>
+                            <span class='text-xs font-semibold uppercase tracking-wide text-primary/80'>Latest news</span>
+                            <h2 class='mt-3 text-xl font-semibold leading-7 text-slate-900'>
+                                <a class='hover:underline' href={`/news/${latestNewsItem.slug}`}>
+                                    {latestNewsItem.title}
+                                </a>
+                            </h2>
+                            <div class='mt-1 text-sm text-slate-500'>By the Pathoplexus Team – {latestNewsItem.date}</div>
```

Comment:
For consistency we should probably get the `websiteName` here instead of writing pathoplexus

Reply:
(Thanks for the review!) I don't think this is a problem (if anything I'd probably hardcode pathoplexus throughout in the pathoplexus repo, but I don't think it's worth the time cost of going through and doing so)